scripts/ldap-access-sync.py

```
@@ -0,0 +1,454 @@
+    attrlist: List[str],
+    page_size: int = 500,
+):
+    """Iterate LDAP search results using RFC 2696 paged results.
+
+    This avoids server-side sizeLimit issues when a subtree is large.
+    """
+    paged = ldap.controls.SimplePagedResultsControl(True, size=page_size, cookie=b"")
+    msgid = client.search_ext(base_dn, scope, filterstr, attrlist=attrlist, serverctrls=[paged])
+
+    while True:
+        rtype, rdata, rmsgid, serverctrls = client.result3(msgid)
```

Ruff (0.15.12)

[warning] Unpacked variable `rtype` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)